src/components/form/Button.jsx

```
@@ -4,6 +4,7 @@ import { useSelector } from "react-redux";
 const Button = ({
   onClick,
   icon: BtnIcon,
+  lastIcon: LastIcon,
```

ESLint

[error] 'lastIcon' is missing in props validation

(react/prop-types)